tests/test_examples.py

```diff
@@ -279,6 +286,7 @@ def test_distributed_inference_examples_stable_diffusion(self):
         testargs = ["examples/inference/distributed/stable_diffusion.py"]
         run_command(self.launch_args + testargs)
 
+    @require_non_hpu  # Error in generate: synNodeCreateWithId failed for node: masked_fill_fwd_i64
```

Comment:
This is the issue you shared [here](https://huggingface.slack.com/archives/C03264192KV/p1739977452256159) on Slack right?